compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -650,7 +660,9 @@ template <typename T> std::map<std::string, int32_t> cook_graph(const T &graph,
     // custom operator
     else
     {
-      auto op_it = std::find(custom_code_vec.begin(), custom_code_vec.end(), operation.type());
+      assert(not operation.custom_code().empty());
+      auto op_it =
+        std::find(custom_code_vec.begin(), custom_code_vec.end(), operation.custom_code());
```

Comment:
can we do this?
```suggestion
      auto op_it =
        std::find(custom_code_vec.begin(), custom_code_vec.end(), op_type);
```

Reply:
actually for better readability, I hope the code be like this without line split
```suggestion
      auto custom_code = operation.custom_code();
      auto op_it = std::find(custom_code_vec.begin(), custom_code_vec.end(), custom_code);
```

Reply:
@seanshpark 

Oh, yes! we can.
Thank you for letting me know. :)